src/tables/post.js

```diff
@@ -50,18 +50,28 @@ function parsePostTable(data, start) {
     return post;
 }
 
-function makePostTable() {
-    return new table.Table('post', [
-        {name: 'version', type: 'FIXED', value: 0x00030000},
-        {name: 'italicAngle', type: 'FIXED', value: 0},
-        {name: 'underlinePosition', type: 'FWORD', value: 0},
-        {name: 'underlineThickness', type: 'FWORD', value: 0},
-        {name: 'isFixedPitch', type: 'ULONG', value: 0},
-        {name: 'minMemType42', type: 'ULONG', value: 0},
-        {name: 'maxMemType42', type: 'ULONG', value: 0},
-        {name: 'minMemType1', type: 'ULONG', value: 0},
-        {name: 'maxMemType1', type: 'ULONG', value: 0}
-    ]);
+function makePostTable(postTable) {
+  const {
+    italicAngle = 0,
+    underlinePosition = 0,
+    underlineThickness = 0,
+    isFixedPitch = 0,
+    minMemType42 = 0,
+    maxMemType42 = 0,
+    minMemType1 = 0,
+    maxMemType1 = 0
+  } = postTable || {};
```

Comment:
Destructuring is a relatively new feature, does our build system correctly convert it to ES5?

Reply:
Yes, it does! I just verified it today when I checked if template literals are converted to string concatenation, which it also does. So we're free to use newer language features (but it doesn't hurt to check from time to time)!